src/main/resources/com/typesafe/sbt/packager/archetypes/scripts/ash-template

```diff
@@ -43,8 +43,12 @@ addApp () {
   app_commands="$app_commands $1"
 }
 
+shellEscape () {
+  printf "'%s'" "$(printf %s "$1" | sed "s/'/'\\\\''/")"
+}
+
 addResidual () {
-  residual_args="$residual_args \"$1\""
+  residual_args="$residual_args $(shellEscape "$1")"
```

Comment:
If I change this line to `residual_args="$residual_args $1"` and get rid of `shellEscape`, then all the tests still pass for me locally. Am I doing something wrong, or is `shellEscape` not actually needed here? I'm running scripted tests with `scripted ash/*`.

Reply:
Hey @dwickern,

I'm afraid that's not what I see on my machine. When I remove the `shellEscape` stuff, the `scripted ash/command-line-settings` test fails. When I put it back in, it succeeds again.